src/lib/KeyManager.ts

```diff
@@ -342,16 +342,21 @@ class KeyManager {
             km = ds.get_key_manager()
           }
           if (km) {
-            resolve(km.get_pgp_fingerprint().toString('hex'));
+            if (km.get_pgp_fingerprint()) {
+              resolve(true)
+            } else {
+              resolve(false)
+            }
+            resolve(km.get_pgp_fingerprint());
           } else {
-            reject(Error('could not verify file'))
+            resolve(false)
```

Comment:
Same here. I thought with async await, we would't need to use CPS.

Reply:
Yes, well technically, we don't actually use CPS here. The resolve method is just for turning the kbpgp method into a promise based call, its not a callback you can pass into the `verifyData` method.